src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int sideA;
+    private int sideB;
+    private int height;
+
+    public IsoscelesTrapezoid(int sideA, int sideB, int height, String color) {
+        super(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.height = height;
+        super.setType("isosceles trapezoid");
+        super.setColor(color);
+    }

```

Comment:
```suggestion
    public IsoscelesTrapezoid(int sideA, int sideB, int height, String color) {
        super(color);
        this.sideA = sideA;
        this.sideB = sideB;
        this.height = height;
        super.setType("isosceles trapezoid");
    }

```

нам не нужно сетить color через сеттер, так как ты уже передал его в родительский конструктор. Исправь, пожаулйста, во всех остальных классах тоже